src/cgame/cg_buildable.cpp

```diff
@@ -2142,8 +2146,22 @@ void CG_DrawBuildableStatus()
 	{
 		cent = &cg_entities[ cg.snap->entities[ i ].number ];
 		es = &cent->currentState;
+		attr = BG_Buildable( es->modelindex );
+
+		if ( es->eType != entityType_t::ET_BUILDABLE || team != attr->team )
+		{
+			continue;
+		}
+
+		bool isBuilder = CG_PlayerIsBuilder( (buildable_t) es->modelindex );
+		bool markedBuilding = es->eFlags & EF_B_MARKED;
+		bool damagedBuilding = ( (float) es->generic1 / (float) attr->health ) < 1;
```

Comment:
Instead of less than 1 could the threshold be 0.7 or something? It's not very concerning if the buildable lost 1 hp; plus it's not a good practice to be comparing a floating point to the exact value of 1.

Reply:
Is 0.99 or 0.95 also ok?
I mean its the whole point to see if health is less than 100%
To see the OM/RC build progress it would be good to get close to full health with the displayed range.

Reply:
I certainly agree that builders should always see the progress bar for a building under construction. But for the case of a damaged building that lost a tiny amount of health, it is not so interesting for players to see. To me it even would be better to see all the health bars rather than only ones less than 99%.

Reply:
Well that's good, the default settings still work like that. And who wants can change it the way they prefer.
I think its useful information to see if a building is attacked, especially for not builders.
To allow more flexibility the threshold could also be made a cvar that leaves this decision to the player.

Reply:
> I think its useful information to see if a building is attacked, especially for not builders.

Yes, it could be useful to see this, in order to know if the building is at risk of getting destroyed or needs repairing. But if you have for example a barb splash attack that does 1 hp of damage to all buildings and you see health bars for all of them, I don't see how this is relevant information. For showing damage I would want to have the threshold significantly lower than 100%.